ContentPatcher/Framework/RawContentPack.cs

```diff
@@ -99,16 +99,6 @@ public bool TryReloadContent([NotNullWhen(false)] out string? error)
             IMigration migrator = new AggregateMigration(content.Format, this.GetMigrations(content));
             if (!migrator.TryMigrateMainContent(content, out error))
                 return false;
-            if (content.Changes.Any())
```

Comment:
This isn't required as it runs in PatchLoader.LoadPatches, so this causes old migrations for content.json to run *after* newer migrations, as they are all now running twice.

Reply:
Fixed upstream now, no longer an issue